Topic5Mysql/MySQLFiles/high-school-DB.sql

```diff
@@ -0,0 +1,169 @@
+UNLOCK TABLES;
+
+--
+-- Table structure for table `notes`
+--
+
+DROP TABLE IF EXISTS `notes`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `notes` (
+  `student_id` int(11) NOT NULL,
+  `course_id` int(11) NOT NULL AUTO_INCREMENT,
+  `partial_note1` decimal(10,0) unsigned NOT NULL,
+  `partial_note2` decimal(10,0) unsigned NOT NULL,
+  `partial_note3` decimal(10,0) unsigned NOT NULL,
+  `final_note` decimal(10,0) unsigned NOT NULL,
+  PRIMARY KEY (`course_id`,`student_id`),
+  KEY `registration_number_idx` (`student_id`),
+  CONSTRAINT `course_id` FOREIGN KEY (`course_id`) REFERENCES `course` (`course_id`) ON DELETE NO ACTION ON UPDATE NO ACTION,
+  CONSTRAINT `student_id` FOREIGN KEY (`student_id`) REFERENCES `student` (`registration_number`) ON DELETE NO ACTION ON UPDATE NO ACTION
+) ENGINE=InnoDB AUTO_INCREMENT=6 DEFAULT CHARSET=utf8;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `notes`
+--
+
+LOCK TABLES `notes` WRITE;
+/*!40000 ALTER TABLE `notes` DISABLE KEYS */;
+INSERT INTO `notes` VALUES (1,1,6,5,4,5),(2,1,6,5,4,5),(3,1,6,5,4,5),(4,1,6,5,4,5),(5,1,6,5,4,5),(6,1,6,5,4,5),(7,1,6,5,4,5),(8,1,6,5,4,5),(9,1,6,5,4,5),(10,1,6,5,4,5),(11,1,6,5,4,5),(12,1,6,5,4,5),(13,1,6,5,4,5),(14,1,6,5,4,5),(15,1,6,5,4,5),(16,1,6,5,4,5),(17,1,6,5,4,5),(18,1,6,5,4,5),(19,1,6,5,4,5),(20,1,6,5,4,5),(1,2,6,5,4,4),(2,2,4,5,2,5),(3,2,9,9,9,10),(4,2,2,2,2,2),(5,2,4,9,2,6),(6,2,4,5,8,6),(7,2,1,8,9,7),(8,2,4,5,4,4),(9,2,4,5,7,6),(10,2,3,3,1,2),(11,2,2,4,6,5),(12,2,5,8,1,4),(13,2,6,7,8,9),(14,2,5,6,3,5),(15,2,5,6,7,10),(16,2,5,2,3,4),(17,2,8,7,8,9),(20,2,4,6,9,8),(24,2,5,5,6,7),(26,2,2,3,1,2),(30,2,1,1,0,1),(1,3,9,9,9,9),(2,3,8,8,8,8),(3,3,7,7,7,7),(4,3,6,5,4,5),(5,3,2,2,2,2),(6,3,4,4,4,4),(7,3,6,5,4,5),(8,3,6,5,4,5),(9,3,6,5,4,5),(10,3,6,5,4,5),(11,3,6,5,4,5),(12,3,6,5,4,5),(13,3,6,5,4,5),(14,3,6,5,4,5),(15,3,6,5,4,5),(16,3,6,5,4,5),(17,3,6,5,4,5),(18,3,6,5,4,5),(19,3,6,5,4,5),(20,3,6,5,4,5),(10,4,6,5,4,5),(11,4,6,5,4,5),(12,4,6,5,4,5),(13,4,6,5,4,5),(14,4,6,5,4,5),(15,4,6,5,4,5),(16,4,6,5,4,5),(17,4,6,5,4,5),(18,4,6,5,4,5),(19,4,6,5,4,5),(20,4,6,5,4,5),(21,4,6,5,4,5),(22,4,6,5,4,5),(23,4,6,5,4,5),(24,4,6,5,4,5),(25,4,6,5,4,5),(26,4,6,5,4,5),(27,4,6,5,4,5),(28,4,6,5,4,5),(29,4,6,5,4,5),(11,5,6,5,4,5),(12,5,6,5,4,5),(13,5,6,5,4,5),(14,5,6,5,4,5),(15,5,6,5,4,5),(16,5,6,5,4,5),(17,5,6,5,4,5),(18,5,6,5,4,5),(19,5,6,5,4,5),(20,5,6,5,4,5),(21,5,6,5,4,5),(22,5,6,5,4,5),(23,5,6,5,4,5),(24,5,6,5,4,5),(25,5,6,5,4,5),(26,5,6,5,4,5),(27,5,6,5,4,5),(28,5,6,5,4,5),(29,5,6,5,4,5),(30,5,6,5,4,5);
+/*!40000 ALTER TABLE `notes` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `schedule`
+--
+
+DROP TABLE IF EXISTS `schedule`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `schedule` (
+  `schedule_id` int(11) NOT NULL AUTO_INCREMENT,
+  `day` varchar(10) NOT NULL,

```

Comment:
You could use an enum here to avoid any issue with the way the name of day is inserted on the DB (uppercase, lowercase, etc)